src/data/matrix/basis.lean

```diff
@@ -20,39 +22,39 @@ open_locale big_operators
 
 variables [decidable_eq l] [decidable_eq m] [decidable_eq n]
 
-variables [semiring α]
+variables [semiring R]
```

Comment:
I'd prefer we leave this as `α` as I intend to generalize these lemmas to non-rings, where I'd like to use `R` in `has_scalar R  α` instead.